parkings/api/enforcement/valid_parking.py

```diff
@@ -33,6 +33,11 @@ def to_representation(self, instance):
         if not instance.is_disc_parking:
             representation.pop('is_disc_parking')
 
+        if instance.time_end is None:
+            replacement_value = getattr(
+                settings, 'PARKKIHUBI_NONE_END_TIME_REPLACEMENT', None)
+            representation['time_end'] = replacement_value
```

Comment:
add `or None`  to this so that it is possible to use empty string in the setting to get this set to None

Reply:
since the setting could be set via env variable, and you can't pass None from there